### PR TITLE
Add data samples to Transfers feature

### PR DESCRIPTION
**Pull request by Sergiodero**
Changes include:

- Adds data samples
- Adds requirements 
- Fixes link to reference.md

### CODE REVIEW

File: docs/getting_started/features/transfers.md [hunk 1]
```diff
@@ -2,6 +2,23 @@
 
 Transfers provide details about transitions between different travel segments (or legs), enabling trip planners to determine the feasibility of journeys that include transfers.
```
Comment (isabelle-dr) [inline docs/getting_started/features/transfers.md]: I think this section in the GTFS Books is a nice addition to the feature description (I have modified it a bit).

> Specifying transfers doesn't imply passengers can't transfer elsewhere. It just shows whether certain transfers are not possible or require a minimum time to transfer.

**Review by isabelle-dr — CHANGES_REQUESTED** [review]
Same meta comments than in the other PRs

- Line break after the ??? note "Sample Data"
- Replace "the following table" with "the following sample"
- Pre-requirements without the "implement" and remove "s"

**Comment by Sergiodero** [comment]
Comments are now addressed, the suggested text was added to the description.

**Review by isabelle-dr — APPROVED** [review]
LGTM!

**Pull request merged**